src/combobox/styles/comboBox.m.css.d.ts

```diff
@@ -1,13 +1,13 @@
-export const clear: string;
-export const clearable: string;
+export const root: string;
 export const controls: string;
-export const disabledOption: string;
-export const dropdown: string;
 export const input: string;
-export const invalid: string;
-export const open: string;
+export const clearable: string;
```

Comment:
Why the churn on these files?  Was it related to a `tcm` run?

Reply:
_sigh_ my editor alphabetizes these automatically, I always forget to disable this feature. I'll blame it on `tcm` just for the heck of it though :)